homeassistant/helpers/trigger.py

```diff
@@ -67,6 +67,19 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+_EXPERIMENTAL_TRIGGER_PLATFORMS = {
+    # "alarm_control_panel",
+    # "assist_satellite",
+    # "climate",
+    # "cover",
+    # "fan",
+    # "lawn_mower",
+    # "light",
+    # "media_player",
+    "text",
+    # "vacuum",
+}
```

Comment:
All except `text` commented out so I didn't have to change all tests.
I'm not sure this set is an OK approach?

Reply:
Yes, it should be explicit, but I was not sure it was OK to hard code it here, or if it should be autogenerated in some fancy way. Let's stick with the hard coded set for now then.